skopt/plots.py

```diff
@@ -137,10 +152,12 @@ def _format_scatter_plot_axes(ax, space, ylabel):
 
                 ax_.xaxis.tick_top()
                 ax_.xaxis.set_label_position('top')
-                ax_.set_xlabel("$X_{%i}$" % j)
+                ax_.set_xlabel(dim_labels[j])
 
-            ax_.xaxis.set_major_locator(MaxNLocator(6, prune='both'))
-            ax_.yaxis.set_major_locator(MaxNLocator(6, prune='both'))
+                if 'log-uniform' == space.dimensions[i].prior:
```

Comment:
to keep with usual python style can you flip the order of the condition to `var == 'value'`, luckily `if var = value` is a syntax error in python